parser/unparser.go

```diff
@@ -37,7 +37,26 @@ import (
 // - Spacing around punctuation marks may be lost.
 // - Parentheses will only be applied when they affect operator precedence.
 func Unparse(expr *exprpb.Expr, info *exprpb.SourceInfo) (string, error) {
-	un := &unparser{info: info}
+	return UnparseFormatted(expr, info)
+}
+
+// TODO
+func UnparseFormatted(expr *exprpb.Expr, info *exprpb.SourceInfo, opts ...UnparserFormattingOption) (string, error) {
```

Comment:
Would it be reasonable to collapse the overloads?

Reply:
Yeah I think so. In this case, the change would be backwards compatible.